src/PerfCounters_aarch64.h

```diff
@@ -0,0 +1,55 @@
+/* -*- Mode: C++; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+// This file is included from PerfCounters.cc
+
+static const char* midr_path =
+    "/sys/devices/system/cpu/cpu0/regs/identification/midr_el1";
```

Comment:
I think this is the point at which we start being concerned about heterogeneous CPUs. On x86 we pin to a random logical CPU that's not already in use. On ARM we could pin to cpu0 for now, but eventually we'll want to at least be able to choose any of the big cores, I think.

Reply:
Yes, I had the same thought while writing this, but I think we should let it be for now and worry about heterogenous CPUs once we've got this a bit more stable on a homogeneous CPU. Heterogeneous CPU support will require a bit of a refactor in general, particularly once you get into things like different ISAs and some cores supporting rr, but some not.